### ISSUE TITLE
Not launching in Ubuntu 20.04 after installing the file klogg-20.10.0.793-Linux.deb

### ISSUE THREAD

**Issue by rohan-paul** [variar/klogg#225]
I installed following the guide from here with the .deb package in Ubuntu 20.04 

``klogg-20.10.0.793-Linux.deb``

But then its just not launching.

**Comment by variar** [comment]
@rohan-paul could you share a link to the guide that was used?

**Comment by rohan-paul** [comment]
@variar I followed the guide from this [repo itself](https://github.com/variar/klogg#installation)

For DEB add Bintray GPG key and repository, then install from apt:

```
apt-key adv --keyserver hkps://keyserver.ubuntu.com --recv-keys 379CE192D401AB61
echo deb https://dl.bintray.com/variar/deb stable utils | sudo tee -a /etc/apt/sources.list
apt-get update
apt-get install klogg
```
I ran again the above command now and **klogg** seems to be installing fine (output in my terminal after running the `sudo apt-get install klogg`

```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
klogg is already the newest version (20.10.0.793).
The following packages were automatically installed and are no longer required:
  python3-click python3-colorama
Use 'sudo apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 35 not upgraded.
```

But then I can not launch klogg in any way, tried both from desktop launcher in Ubuntu 20.04 and also from the right-click context menu of a .csv file.

**Comment by variar** [comment]
Could you try to start klogg from the terminal with `/usr/bin/klogg -d` command? That should produce some verbose output. There might be some clues indicating root cause of the issue.

**Comment by variar** [comment]
Looks like you've installed deb-package from CI builds. They are not included into bintray repository. Did you install deb package manually with `sudo dpkg -i /path/to/deb/file`? After such installation in some cases one needs to run `sudo apt-get install -f` to fix dependencies.

**Comment by variar** [comment]
Tried in docker container. Got error:
```
/usr/bin/klogg: symbol lookup error: /usr/bin/klogg: undefined symbol: _ZdlPvm, version Qt_5
```

Could be caused by incompatible compiler flags.

**Comment by variar** [comment]
Should be fixed in 20.10.0.797. @rohan-paul could you check new build?

**Comment by rohan-paul** [comment]
Thanks now its working great.
